### ISSUE TITLE
REQUEST/ISSUE: cursor can click on minimap

### ISSUE THREAD

**Issue by BasicsOnly** [CptWesley/NewWorldMinimap#32]
A big problem with this minimap during combat is that the cursor (usually invisible in FS mode) will click on the minimap while attacking and mess up combat interactions. Is there a way to make the minimap non-clickable/selectable with left click during always on top?

**Comment by rafek1241** [comment]
If you want to make it like this you will need to create two forms and shuffle them. 
1. Inactive window - visible when hotkey clicked that is totally not interactive (except hotkey).
2. Active window - with interactivity.

**Comment by CptWesley** [comment]
No, I just needed to set the window properties using the `SetWindowLongPtr` native Windows API to indicate that the window should not receive any click events, meaning they go to whatever window is underneath.
